code/game/objects/items/plantable_flags.dm

```diff
@@ -1,60 +1,225 @@
-/obj/item/flag_base
-	name = "basic flag"
-	desc = "It's a one time use flag built into a telescoping pole ripe for planting."
-	icon = 'icons/obj/items/plantable_flag.dmi'
-	icon_state = "flag_collapsed"
-	force = 3
-	throwforce = 2
+///Range of the aura
+#define FLAG_AURA_RANGE 10
+///Range of the aura when deployed
+#define FLAG_AURA_DEPLOYED_RANGE 14
+///The range in tiles which the flag makes people warcry
+#define FLAG_WARCRY_RANGE 5
+///Strength of the aura
+#define FLAG_AURA_STRENGTH 3
+///Penalty aura for losing or otherwise disgracing the flag
+#define LOST_FLAG_AURA_STRENGTH -2
+
+/obj/item/plantable_flag
+	name = "\improper TerraGov flag"
+	desc = "A flag bearing the symbol of TerraGov. It flutters in the breeze heroically. This one looks ready to be planted into the ground."
+	icon = 'icons/obj/items/flags/plantable_flag_large.dmi'
+	icon_state = "flag_tgmc"
+	inhand_x_dimension = 64
+	inhand_y_dimension = 64
+	worn_icon_list = list(
+		slot_l_hand_str = 'icons/mob/inhands/items/large_flag_left.dmi',
+		slot_r_hand_str = 'icons/mob/inhands/items/large_flag_right.dmi',
+	)
+	w_class = WEIGHT_CLASS_HUGE
+	force = 55
+	attack_speed = 15
+	attack_verb = list("stabbed", "thrust", "smashed", "thumped", "bashed", "attacked", "clubbed", "speared", "jabbed", "torn", "gored")
```

Comment:
Torn? Isn't this grammatically incorrect

Reply:
Isn't it? Its used by like every melee weapon. 
![image](https://github.com/user-attachments/assets/4abb3e25-b839-4125-9a7e-0902fa3f0153)

Reply:
shrug